govcd/lbserverpool_test.go

```diff
@@ -110,6 +110,11 @@ func (vcd *TestVCD) Test_LBServerPool(check *C) {
 
 	check.Assert(createdLbPool.Algorithm, Equals, lbPoolConfig.Algorithm)
 
+	// GetLbServerPools should return at least one pool which is our.
+	pools, err := edge.GetLbServerPools()
+	check.Assert(err, IsNil)
+	check.Assert(pools, Not(HasLen), 0)
```

Comment:
We should check that at least some of the server fields are as expected.

Reply:
The problem is that if the testing environment is not clean (e.g. there is more than 1VS/Pool/etc.) we cannot be sure what are correct values. Generally we should not expect correct values, since we're not filtering by anything - we want to get everything without expecting something specific (for which there is get by id, get by name funcs)